core/image.cpp

```diff
@@ -1680,7 +1680,7 @@ Error Image::generate_mipmap_roughness(RoughnessChannel p_roughness_channel, con
 				int pixel_ofs = y * w + x;
 				Color c = _get_color_at_ofs(ptr, pixel_ofs);
 
-				float roughness;
+				float roughness = 0;
```

Comment:
This is a false positive as we handle all possible enum values in the switch... unless `p_roughness_channel` can be passed with a value outside the enum, in which case this is correct, but should we then add a `default: ERR_FAIL_V("Bug");` to all our switches...?